wiki/content/enterprise-features/index.md

```diff
@@ -703,3 +702,47 @@ Here's an example to run bulk loader with a key used to write encrypted data:
 ```bash
 dgraph bulk --encryption_key_file "./enc_key_file" -f data.json.gz -s data.schema --map_shards=1 --reduce_shards=1 --http localhost:8000 --zero=localhost:5080
 ```
+
+## Encrypted Backups
+
+Encrypted backups are a Enterprise feature that allow you to encrypt your backups and restore them. This documentation describes how to implement encryption into your binary backups
+
+### New flag “Encrypted” in manifest.json
+
+A new flag “Encrypted” is added to the `manifest.json`. This flag indicates if the corresponding binary backup is encrypted or not. To be backward compatible, if this flag is absent, it is presumed that the corresponding backup is not encrypted.
+
+For a series of full and incremental backups, per the current design, we don't allow mixing of encrypted and unencrypted backups. As a result, all the full and incremental backups must be all encrypted or not. This flag helps with checking this restriction.
+
+NOTE: If a mismatch is found (e.g. latest backup is encrypted but current alpha is not configured with encryption or vice-versa), an error is thrown. The user must then force a full backup using the `forcefull` flag in the backup API to force a backup.
+
+### AES And Chaining with Gzip
```

Comment:
This section is an implementation detail. I dont think we need to document this.